src/lib/booleans.js

```diff
@@ -0,0 +1,131 @@
+function negate(a) {
+  // return !a - to return opposite of a
+  if (a === true) {
+    return false;
+  }
+  return true;
+};
+
+function both(a, b) {
+  if (a && b === true) {
+    return true;
+  }
+  return false;
+}
+
+function either(a, b) {
+  if (a || b === true) {
+    return true;
+  }
+  return false;
+};
+
+function none(a, b) {
+  if (a === false && b === false){
+    return true;
+  }
+  return false;
+};
+
+function one(a, b) {
+  if (a === true && b === false) {
+    return true;
+  }
+  if (a === false && b === true) {
+    return true;
+  }
+  return false;
+};
+
+function truthiness(a) {
+  if (a) {
+    return true;
+  }
+  return false;
```

Comment:
The logic here is fine, but it can be condensed a lot using two NOT operators (!!) together to create a double negation. What this does is to first evaluate the value to a boolean that will be the opposite of what it actually is, eg passing in a string (which is a truthy value) will be made `false` by one of the ! operators, then the second ! will flip that to `true`, and so evaluating the string as being truthy. Nice trick that gets used more often than you might think!
```
  return !!a
```